packages/off-chain/shell/run_bacalhau.sh

```diff
@@ -0,0 +1,120 @@
+# shell script to run docker image on bacalhau
```

Comment:
you're really getting into the shell scripting, aren't you? :D

Reply:
yes, after the intel vs apple issue, it's easier to write script which runs for all rather than writing docs that people can refer.